extensions/placeholder/src/components/action-on-placeholder-image.tsx

```diff
@@ -0,0 +1,83 @@
+import { Action, ActionPanel, Icon } from "@raycast/api";
+import StylizePlaceholderImage from "../stylize-placeholder-image";
+import { setWallpaper } from "../utils/common-utils";
+import { PicsumImageAction } from "./picsum-image-action";
+import { ActionOpenPreferences } from "./action-open-preferences";
+import React, { Dispatch, SetStateAction } from "react";
+import { PicsumImage } from "../types/types";
+import { Preferences } from "../types/preferences";
+import { ImageDetail } from "./image-detail";
+import { RevealImageAction } from "./reveal-image-action";
+
+export function ActionOnPlaceholderImage(props: {
+  picsumImage: PicsumImage;
+  preferences: Preferences;
+  page: number;
+  setPage: Dispatch<SetStateAction<number>>;
+}) {
+  const { picsumImage, preferences, page, setPage } = props;
+  return (
+    <ActionPanel>
+      <Action.Push
+        icon={{
```

Comment:
Hi @mattisssa 
> Hey, thanks for the update! I find one small thing, can we add an icon here to match the Store Guidelines? And we good to go.

There is an icon here, but I don't know if it is not displayed on your device, I am displaying it normally here

> I also found crazy performance issues in your extension and though that you could probably use a smaller size for images in Grid.Item and for export use the original sizes. Would be nice.

Thank you for your suggestion, I will make changes, it will become more beautiful and better performance

Reply:
>There is an icon here, but I don't know if it is not displayed on your device, I am displaying it normally here

Can this be the theme issues or something? I'll take a look later

Reply:
I just opened the command and noticed that this action also does not have an icon, but it was quickly displayed. Maybe it takes time to load the icons in local assets?